metadata.yaml

```diff
@@ -50,7 +50,7 @@ resources:
     type: oci-image
     description: Prometheus exporter for apache
     auto-fetch: true
-    upstream-source: bitnami/apache-exporter:0.11.0
+    upstream-source: bitnami/apache-exporter:0.13.3
```

Comment:
Similar to other PR, are there any trivy ignores we could remove now?

Reply:
Only images resulting from ROCKS or from Dockerfiles are scanned by Trivy. External ones set on upstream-source in metadata file are not scanned.